build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -1,7 +1,7 @@
 #!/usr/local/bin/ruby
 require_relative "ConfigParseErrorLogger"
 
-@td_agent_bit_conf_path = "/etc/opt/microsoft/docker-cimprov/td-agent-bit.conf"
+@td_agent_bit_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit.conf"
```

Comment:
lets rename td_agent_bit -> fluent_bit so we have everything consistent new name.